woocommerce/woocommerce-ios
# Order Fulfillment: Add store country at the beginning of the list of shipment providers

Implements second part of #847 

This would be result for a HK-based store (there are no tracking providers for it)
![Simulator Screen Shot - iPhone 8 - 2019-05-06 at 16 47 12](https://user-images.githubusercontent.com/2722505/57215528-bab17580-701f-11e9-80fb-b8717a0def7b.png)

And this would be the result for a US-based store:
![Simulator Screen Shot - iPhone 8 - 2019-05-06 at 16 47 49](https://user-images.githubusercontent.com/2722505/57215547-c3a24700-701f-11e9-85c5-c0fd2f7a3217.png)

## Changes
- Added a class to map the country code provided as a `SiteSetting` to a readable country name, in order to match it with the current list of tracking providers. 
- Add logic to `ShippingProvidersViewModel` to split the data fetch in two results controllers, one for countries excluding the store country, another for providers in the country of the store.
- Add logic to present that data.

## Testing
- Navigate to an order > Fulfill order > Add tracking
- Check that the providers that correspond to the store country show up at the top of the list
- Check that, if there are no providers corresponding to the store country, the list renders alphabetically.
- Check that any provider can be selected, and shipment tracking can be created.

Update release notes:
